tests/error-test.conf

```diff
@@ -0,0 +1,26 @@
+
+
+# FIXME: THIS FILE SHOULD NOT BE MERGED TO DEVELOP
```

Comment:
do we need these FIXME if we're keeping this now?

Reply:
I'll replace it with an explanatory comment.